resources/views/components/argument-card/card.blade.php

```diff
@@ -51,7 +51,7 @@ class="{{ $isEditing?->is($argument) ? 'hover:text-green-800' : 'hover:text-blue
 
         <x-argument-card.card-footer
             :argument="$argument"
-            :user="$user"
+            :user="$argument->user"
```

Comment:
Could you remove this line? It's been fixed in another way. Having this line actually causes a bug somewhere else.

Reply:
sure, Now it's back to the way it was.